validator/validators/performance/performance_models.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package performance
+
+type PerformanceInformation map[string]interface{}
```

Comment:
nit: rename TestResult or TestInfo. The package is already named performance, so you don't really need it in the name.

Reply:
Question: if we know the properties, why is PerformanceInformation unstructured?

Reply:
The reason why it would be structured this way [is because](https://github.com/aws/amazon-cloudwatch-agent-test/pull/153/files#diff-334efd747575193d3505843e769add29bb2818c73a18ca77c9c4f97105ec3104R120)
```
func ReplaceItemInDatabase(databaseName string, packet map[string]interface{}) error {
	item, err := attributevalue.MarshalMap(packet)
	if err != nil {
		return err
	}

	_, err = DynamodbClient.PutItem(ctx,
		&dynamodb.PutItemInput{
			Item:      item,
			TableName: aws.String(databaseName),
		})

	return err
}
```
since you can argue it would be better to use struct for specific test but for general case, it would be better to use `map[string]interface{}`